### DISCUSSION TITLE
v4 7-Slot

### DISCUSSION THREAD

**Discussion by ImStuBTW** [sanni/cartreader#365] [General Chat]
I was about to submit an order for the PCBs and parts needed to make a v4 1-Slot, when I noticed the documentation had been updated with the new v4 7-Slot design. It looks nice! I was already going to place a pretty maximalist build, and I like having the option to built everything out as a single self-contained unit. It looks like the documentation's still being updated, so I hope you don't mind if I ask a few questions about the design.

1. Was there any particular impetus to move from the adapter-style approach back to a multi-slot design?
2. The build instructions mention soldering a 1x8 female header onto the six-slot PCB. This doesn't appear to connect to anything at the moment? The PCB labels these pinouts as snesCIC. Is there going to be a breakout board for that IC at some point?
3. On the one-slot N64 adapter, there's two through hole pins for the 1K resistor, and then three pins for the N64 controller port. On the six-slot PCB, there's two through hole pins for the 1K resistor, and then GND and VCC pins for the controller port. Is the idea that the user would need to put the N64 controller port's DAT wire and the 1K resistor through the same through hole?
4. The back plate is 3mm thick. If the LCD cables are 2mm too thick, would it be possible to make a 2mm notch in the backplate for the cables and leave the stock connectors alone?
5. Finally, how is voltage selection performed? If I understand correctly, on the one-slot design the adapter PCB routed the correct 3v or 5v to the Arduino for whichever adapter you had plugged in. The six-slot PCB does not seem to have any voltage selection switches on it. Have you done some interesting trick to provide the proper power where it needs to go, or does the GB/GBC/GBA voltage switch toggle power for everything now?

Thanks for all the hard work you've put into this cart reader and the new designs!

**Comment by sanni** [comment]
I like questions/feedback, it always helped the project to move forward. So thanks for asking them. 👍 

1. Just convenience for people that want to dump multiple game systems. I noticed having more than three adapters starts to get tedious.
2. Exactly, but I have not made the small PCB yet. It also allows for using different CIC chips e.g. through hole instead of SMD.
3. Yes the data wire gets soldered to the middle pin of the three that also connects to the resistor. 
4. That would be an alternative, it also might just fit as-is without filing the connector down if you bend it a little.
5. The GBA slot adapter switches the voltage for the whole Cart Reader.

**Reply by ImStuBTW** [reply]
Thanks, this helps a lot!

**Comment by sanni** [comment]
I will also release the V5 main PCB soon. It will use the same six-slot-adapter but it will remove the battery and instead integrate the GBA slot.

V5 with six slot adapter:
![cart_reader_hw5](https://user-images.githubusercontent.com/19169494/169024346-9eb0011e-c0f8-4c74-a5a2-0e52cdca8d67.jpg)

V5 with six slot adapter but same height as V4:
![cart_reader_hw5_2](https://user-images.githubusercontent.com/19169494/169024347-8766ec76-52b6-48ae-9581-447b8fee30d4.jpg)

current V4 with six slot adapter:
![cart_reader_hw4](https://user-images.githubusercontent.com/19169494/169024342-8cced31e-f220-47ba-b4cd-4bd21cdfd850.jpg)

The idea behind V5 will be to make it even more simple to build.

**Reply by ImStuBTW** [reply]
Oh wow, looks great! The battery module output 3v & 5v power previously, right? How are you going to handle the voltage switching on this new board?

**Comment by sanni** [comment]
I only used the 5V Output from the battery and the LCD module did the 5V to 3.3V conversion. 
The V5 will have two switches, 3V/5V and ON/OFF.

The V4 with six-slot-adapter has basically the same thing, ON/OFF switch on the battery module and 3V/5V switch on the GBA adapter PCB (which switches everything between 5V and 3V not just the GBA slot)

The original V4 with the single slot cartridge adapters used the wiring on the slot adapters to switch between 3V/5V.
